### PR TITLE
Website updates: Release 2.4

### PR DESCRIPTION
**Pull request by dsarkar**
**MERGE when R 2.4 roll out starts. Monitor Internal Tracking ID: [EXPOSUREAPP-7321](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7321)**

Updates:
- COMPLETED: Privacy notice - in this branch: dsarkar-release-2-4 -
- COMPLETED: Screenshots - separate branch, base branch: dsarkar-release-2-4 - MERGE THIS PR https://github.com/corona-warn-app/cwa-website/pull/1394 FIRST then https://github.com/corona-warn-app/cwa-website/pull/1393
- COMPLETED: FAQs  - separate branch, base branch: dsarkar-release-2-4 -MERGE THIS PR https://github.com/corona-warn-app/cwa-website/pull/1404 FIRST then https://github.com/corona-warn-app/cwa-website/pull/1393

### CODE REVIEW

**Comment by Ein-Tim** [comment]
Don't forget to update the current app versions to 2.4.2 (or whatever will actually be released) before merging this.

**Comment by Ein-Tim** [comment]
Version 2.4.1 has been released to Android & iOS: https://github.com/corona-warn-app/cwa-app-ios/releases/tag/v2.4.1 & https://github.com/corona-warn-app/cwa-app-android/releases/tag/v2.4.1.

Please update the gulpfile accordingly and merge this PR.

File: gulpfile.js [hunk 1]
```diff
@@ -354,10 +354,10 @@ function replaceVersionNumbers() {
     .src([PATHS.dist + "/**/*.html"])
     .pipe(replace('[ios.latest-os-version]', '14.6'))
     .pipe(replace('[ios.minimum-required-os-version]', '12.5'))
-    .pipe(replace('[ios.current-app-version]', '2.3.3'))
+    .pipe(replace('[ios.current-app-version]', '2.4'))
```
Comment (Ein-Tim) [inline gulpfile.js:357]:
```suggestion
    .pipe(replace('[ios.current-app-version]', '2.4.1'))
```

File: gulpfile.js [hunk 2]
```diff
@@ -354,10 +354,10 @@ function replaceVersionNumbers() {
...
     .pipe(replace('[android.latest-os-version]', '11'))
     .pipe(replace('[android.minimum-required-os-version]', '6'))
-    .pipe(replace('[android.current-app-version]', '2.3.4'))
+    .pipe(replace('[android.current-app-version]', '2.4'))
```
Comment (Ein-Tim) [inline gulpfile.js:360]:
```suggestion
    .pipe(replace('[android.current-app-version]', '2.4.1'))
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
To make your work a bit easier (:

**Comment by dsarkar** [comment]
@Ein-Tim Don’t know why, but I couldn’t commit your changes . Will update it later . Thanks anyway! Good night !

**Pull request merged**